src/components/App.jsx

```diff
@@ -26,54 +23,51 @@ class App extends React.Component {
     } else {
       id = window.location.pathname.slice(1, 4);
     }
-    console.log(id);
+
     $.ajax({
       context: this,
       method: 'GET',
       url: `/products/${id}`,
       success: function(result) {
         var products = JSON.parse(result[0]);
         var shops = JSON.parse(result[1]);
```

Comment:
I dont want to be a broken record here but these parses would also probably be unnecessary if you didn't stringify them initially

Reply:
I always thought I had to stringify it and parse it to convert it from data packet to JSON, but I will try it without now that you said something about it!